README.md

```diff
@@ -1 +1,99 @@
-# hdfs-group-mapper
\ No newline at end of file
+# Stackable Group Mapper for Apache Hadoop
+
+HDFS internally uses user groups for group permissions on folders etc. For this reason it is not enough to just have the groups in OPA during authorization, but they actually need to be available to Hadoop. Hadoop offers a few default group providers, such as:
```

Comment:
This should at least once mention what OPA means for someone stumbling over this and probably should also include a single introduction sentence. I'm only on my phone but I can provide an example tomorrow if I don't forget